src/Authorizer.ts

```diff
@@ -93,7 +92,14 @@ export class Authorizer {
         this.enforcer = await casbin.newEnforcer(m);
         if ('p' in obj) {
             for (const sArray of obj['p']) {
-                await this.enforcer.addPolicy(sArray[1].trim(), sArray[2].trim(), sArray[3].trim());
+                let arr = sArray as string[];
+                arr = arr.map(v => v.trim())
+                const pType = arr.shift()
+                if (pType == 'p'){
+                    await this.enforcer.addPolicy(...arr);
+                } else if (pType == 'g'){
```

Comment:
As for now, node-casbin has supoorted multipie Role Manager. It means we will also has `g2,g3...` policies.
It's OK for just supoort one role manager. But plz add a warning when the user got this unhandle situation.

Reply:
Thank you for your suggestion, I will finish it as soon as possible in the next iteration